frontend/src/views/Project/RolePage/components/RolePermissionsSection/RolePermissionsSection.tsx

```diff
@@ -1,127 +1,52 @@
-import { useForm } from "react-hook-form";
+import { FormProvider, useForm } from "react-hook-form";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { zodResolver } from "@hookform/resolvers/zod";
+import { AnimatePresence } from "framer-motion";
```

Comment:
Not able to save for the first policy?
Uploading Screen Recording 2024-09-26 at 11.16.31 PM.mov…